frontend/app/components/Spots/SpotsList/SpotListItem.tsx

```diff
@@ -26,9 +28,12 @@ interface ISpotListItem {
 }
 
 function SpotListItem({ spot, onRename, onDelete, onVideo, onSelect }: ISpotListItem) {
-  const [isEdit, setIsEdit] = React.useState(false)
+  const [isEdit, setIsEdit] = useState(false);
+  const [loading, setLoading] = useState(true);
```

Comment:
unused?

Reply:
@nick-delirium  What exactly are you referring to? not using React.hook or loading state?

Reply:
loading state.